server/app.js

```diff
@@ -1,24 +1,44 @@
-const express = require('express')
-const favicon = require('serve-favicon')
-const logger = require('morgan')
-const cookieParser = require('cookie-parser')
-const bodyParser = require('body-parser')
+const express = require( 'express' )
+const favicon = require( 'serve-favicon' )
+const logger = require( 'morgan' )
+const cookieParser = require( 'cookie-parser' )
+const bodyParser = require( 'body-parser' )
+const session = require( 'express-session' )
+const pgSession = require( 'connect-pg-simple' )
 
-const users = require('./routes/users')
+const users = require( './routes/users' )
 
 const protect = require( './middleware/protect' )
-const errorHandler = require('./handlers/serverError')
-const notFoundHandler = require('./handlers/notFound')
+const errorHandler = require( './handlers/serverError' )
+const notFoundHandler = require( './handlers/notFound' )
 
 const app = express()
 
-app.use(logger('dev'))
-app.use(bodyParser.json())
-app.use(bodyParser.urlencoded({ extended: false }))
-app.use(cookieParser())
+app.use( logger( 'dev' ))
+app.use( bodyParser.json())
+app.use( bodyParser.urlencoded({ extended: false }))
+app.use( cookieParser())
+app.use( session({
+  store: new ( pgSession( session ))({
+    conObject: {
+      host: process.env.DATABASE_HOST,
+      port: process.env.DATABASE_PORT,
+      //TODO: this changes on production
+      database: process.env.DATABASE_NAME_DEV
+    }
+  }),
+  cookie: {
+    maxAge: 30 * 24 * 60 * 60 * 1000
+  },
+    secret: process.env.SECRET,
+    saveUninitialized: true,
+    resave: false,
+    name: "user_id"
+  }))
 
-app.use('/users', users)
-app.get( '/protected-test', protect, (request, response) => {
+app.use( '/users', users )
+app.get( '/protected-test', protect, ( request, response ) => {
+  console.log(request.session)
```

Comment:
Did you mean to keep this console.log?

Reply:
Probably not. I'll add it back in for testing purposes later!